src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -15,17 +15,28 @@
         event.preventDefault();
 
         const config = JSON.parse(event.currentTarget.dataset.udwConfig);
+        const confirmLabel = Translator.trans(/*@Desc("Add locations")*/ 'confirm.label', {}, 'admin_ui_admin_limitation_pick');
+        const title = Translator.trans(/*@Desc("Choose locations")*/ 'title', {}, 'admin_ui_admin_limitation_pick');
```

Comment:
@webhdx In this repository I left prefix `admin_ui` which works in my opition in the same way as prefix `mfu` or prefix `udw` in `admin-ui-modules`, what do you think?

Reply:
It's going to make a huge mess:
![image](https://user-images.githubusercontent.com/10212760/42636440-1e0f6b72-85e9-11e8-8538-cefe5f3298e8.png)

After this PR we are going to have additional 10 new files. I don't think there is a need to go as granular, `universal_discovery_widget` and `multi_file_upload` domains are fine. If you need more separation you can play with translation keys instead. Also please check if we already have suitable domains first.

Reply:
We - me @sunpietro and @dew326 - agreed in private that we want translations per component.